frontend/components/design/Main.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <main-visual />
+  <div class="bg-yellow-900">
+    <div class="container mx-auto">
+      <Sentence />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
```

Comment:
please use `<script setupup />`, you can find  links in https://github.com/rainy-me/tsukiyo/blob/d351653bc83e1ed89ed56106d8c02c96f945d1db/frontend/admin/EventEditor.vue#L37